### PR TITLE
Adding support of SAP Defined Specification Extensions using  @OpenAPI.Extensions

### PR DESCRIPTION
**Pull request by RoshniNaveenaS**
In this PR, adding documentation for  @OpenAPI.Extensions

### CODE REVIEW

File: advanced/publishing-apis/openapi.md [hunk 1]
```diff
@@ -202,6 +202,22 @@ annotate SampleService with @(
 );
 ```
 
+## OpenAPI Extensions
```
Comment (renejeglinsky) [inline advanced/publishing-apis/openapi.md:205]: Why not include it in the previous section with a new row to the table and an extended example?

Comment (RoshniNaveenaS) [reply advanced/publishing-apis/openapi.md]: Have made the changes suggested

**Comment by RoshniNaveenaS** [comment]
Hello @renejeglinsky ,
I am not sure why syntax error is shown here. I checked by having a CDS file with the same lines, it seemed to be working without any error. Can you please guide me to know what am I missing?

Thanks in advance!

File: advanced/publishing-apis/openapi.md [hunk 2]
```diff
@@ -183,26 +183,35 @@ annotate MyService with @(
...
 This is an example of a CDS service annotated with the annotations above:
 
 ```cds
 annotate SampleService with @(
     OpenAPI:{
-        externalDocs:{
+        externalDocs: {
             description: 'API Guide',
             url        : 'https://help.sap.com/docs/product/sample.html'
         }
```
Comment (renejeglinsky) [inline advanced/publishing-apis/openapi.md:203]:
```suggestion
        },
```

File: advanced/publishing-apis/openapi.md [hunk 3]
```diff
@@ -183,26 +183,35 @@ annotate MyService with @(
...
+        },
+        Extensions: {
+        ![compliance-level]: 'sap:base:v1'
+      }
     }
 );
 ```
 
 
+`Extensions` annotation can be used in root, entity and in function/action level.
```
Comment (renejeglinsky) [inline advanced/publishing-apis/openapi.md:212]: Can this sentence move to the table?

File: advanced/publishing-apis/openapi.md [hunk 4]
```diff
@@ -183,21 +183,27 @@ annotate MyService with @(
 ```
 [See it in context.](https://github.com/chgeo/cds-swagger-ui-express/blob/e5794c55b53dd3e43ebe8ffcfff69341b6eac9c7/tests/app/services.cds#L23-L34){.learn-more}
 
+
 ## OpenAPI
 
 | Term              | Annotation Target | OpenAPI field                                                                  |
 |-------------------|-------------------|--------------------------------------------------------------------------------|
 | `externalDocs`  | EntityContainer   | Links to external documentation that explain more about APIs are helpful to developers. |
+| `Extensions` | EntityContainer   | To add the sap defined (`x-sap`) Specification Extensions. This annotation can an be used in root, entity and in function/action level.                                        |
```
Comment (renejeglinsky) [inline advanced/publishing-apis/openapi.md:192]: Is uppercase needed here for some reason?

```suggestion
| `Extensions` | EntityContainer   | To add the sap defined (`x-sap`) specification extensions. This annotation can an be used in root, entity and in function/action level.                                        |
```

**Comment by RoshniNaveenaS** [comment]
Hello @renejeglinsky,
Have made the changes

**Review by renejeglinsky — APPROVED** [review]

**Pull request merged**